app/assets/src/components/views/report/ReportTable/constants.js

```diff
@@ -0,0 +1,50 @@
+const DOC_LINK =
+  "https://github.com/chanzuckerberg/idseq-dag/wiki/IDseq-Pipeline-Stage-%233:-Reporting-and-Visualization#reports";
+
+export const REPORT_TABLE_COLUMNS = {
+  NT_aggregatescore: {
+    title: "Aggregate score",
+    tooltip:
+      "Experimental ranking score for prioritizing microbes based on abundance within the sample (rPM) as well as compared to control samples (Z-score).",
+    link: DOC_LINK,
+  },
+  zscore: {
+    title: "Z-score",
+    tooltip:
+      "Statistic used for evaluating the prevelance of microbes in your sample as compared to background contaminants.",
```

Comment:
super-nit: I think we should stick with `the sample` to be consistent (ex https://medium.com/@jsaito/is-this-my-interface-or-yours-b09a7a795256)